test/conf/cassandra-sslcontextfactory.yaml

```diff
@@ -67,6 +67,8 @@ server_encryption_options:
     internode_encryption: none
     keystore: conf/.keystore
     keystore_password: cassandra
+    outbound_keystore: conf/.keystore
```

Comment:
Is the outbound keystore a required configuration? If not, can we have an example without it? We should make sure about the backwards compatibility.

Reply:
@maulin-vasavada We will maintain backward compatibility, the default value to the `outbound_keystore` is the `keystore` (like we have currently)